opam

```diff
@@ -7,21 +7,15 @@ bug-reports: "https://github.com/mirage/mirage-bootvar-xen/issues/"
 dev-repo: "https://github.com/mirage/mirage-bootvar-xen.git"
 license: "ISC"
 build: [
-  ["ocaml" "setup.ml" "-configure" "--disable-tests"]
-  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "pkg/pkg.ml" "build"]
 ]
-build-test: [
-  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
-  ["ocaml" "setup.ml" "-build"]
-  ["ocaml" "setup.ml" "-test"]
-]
-install: [make "install"]
 remove: [
   ["ocamlfind" "remove" "mirage-bootvar"]
+  ["ocamlfind" "remove" "mirage-bootvar-xen"]
 ]
```

Comment:
I convinced myself that there's minimal chance of breakage if we don't include a `remove` line for `mirage-bootvar`, so that seems fine.  Thanks.